src/mixins/breakpoints/index.ts

```diff
@@ -4,16 +4,25 @@ import {
   landingMobileSizeBreak,
   mobileSizeBreak,
   mobileSmallSizeBreak,
+  mobile360Break,
 } from './const';
 export * from './const';
 
 /** `max-width: ${mobileSmallSizeBreak}px` */
-export const mobileSmallOnly = (cssContent: string) => css`
+export const mobileSmallOnly = (cssContent: FlattenSimpleInterpolation) => css`
   @media only screen and (max-width: ${mobileSmallSizeBreak}px) {
     ${cssContent}
   }
 `;
 
+/** `max-width: ${mobile360Break}px` */
+export const mobile360SizeOnly = (
```

Comment:
```suggestion
export const mobile360pxOnly = (
```
px에 이미 size라는 의미가 내포되어 있어서 요건 어떤가 싶어요
하지만 다른 친구들한테 size가 이미 사용되어 있군요 이건 그냥 댓글만 남겨두겠습니다

Reply:
그렇네요 꼭 컨벤션을 맞춰야하나? 생각하는데 맞출 필요 없는 것 같아요ㅋㅋ 종택님 의견에 따르겠습니다!